runtime/src/sandbox.c

```diff
@@ -218,7 +213,7 @@ sandbox_free(struct sandbox *sandbox)
 
 	/* Free Sandbox Struct and HTTP Request and Response Buffers */
 
-	if (likely(sandbox->stack.buffer != NULL)) sandbox_free_stack(sandbox);
+	if (likely(sandbox->stack->buffer != NULL)) sandbox_free_stack(sandbox);
```

Comment:
Adding a note for myself. This should be `if (likely(sandbox->stack->buffer != NULL))` now that the stack is its own heap allocation.